src/pages/send/multi-send/multi-send.ts

```diff
@@ -224,8 +226,9 @@ export class MultiSendPage extends WalletTabsChild {
         recipient
       });
     } else {
+      const address = this.incomingDataProvider.extractAddress(this.search);
```

Comment:
Maybe we should change the name of the function `extractAddress`, since it only extracts the address for eth, right?

Reply:
Changed function to extract address for all URIs regardless of coin. Ex:

Replaces `prefix:` and any `?param=123` or `&param=123`.